src/components/CCPDashboard/CCPDashboardPage.tsx

```diff
@@ -98,17 +114,62 @@ const TabPanel = (props: TabPanelProps) => {
   );
 };
 
+type LocationState = { numUpdates: number };
+
 const CCPDashboardPage = ({ match }: RouteComponentProps<TParams>) => {
   const classes = useStyles();
+  const location = useLocation<LocationState>();
+
   const { eventId, ccpId, patientId } = match.params;
+  const { numUpdates } = location.state || { numUpdates: 0 };
+
+  const [lastNumUpdates, setLastNumUpdates] = React.useState(numUpdates);
```

Comment:
Potentially nitpick: does `lastNumUpdates` refer to the previous `numUpdates`? If it does, `prevNumUpdates` may be more clear. (But also this is just my personal opinion)

Reply:
i think `prev...` makes more sense in this context too!